src/Compilers/Core/Portable/Operations/Operation.cs

```diff
@@ -17,8 +17,7 @@ namespace Microsoft.CodeAnalysis
     /// </summary>
     internal abstract class Operation : IOperation
     {
-        protected static readonly IOperation s_unset = new EmptyOperation(
-            semanticModel: null, syntax: null, type: null, constantValue: null, isImplicit: true);
+        protected static readonly IOperation s_unset = new EmptyOperation(semanticModel: null, syntax: null!, isImplicit: true);
```

Comment:
>null! [](start = 103, length = 5)

Should we use some sentinel instead?

Reply:
These nodes themselves are only used as sentinels for lazy operation children. They'll be going away eventually in this series of changes, so I don't think it's important.